src/honeybadger.cr

```diff
@@ -33,6 +34,10 @@ module Honeybadger
     # False = never report data
     property report_data : Bool? = nil
 
+    # Merge the log context into the Honeybadger payload
+    # before sending errors to the reporting API.
+    property merge_log_context : Bool = true
```

Comment:
This is the configuration option which can disable log context data ingestion. I set the default value to be true here because it seems like that's what team Honeybadger would want. This means that any usage of `Log.context.set` already existing in a codebase will automatically populate, so this shard could be more useful straight out of the box -- I'm unclear if the crystal community at large really even knows this feature exists in the Log class.

Reply:
This seems like a reasonable default to me. Log context is a neat feature.

Reply:
I love this idea. Doesn’t _force_ you to use `Log.context`, but it accommodates it if you choose to. 🙌🏼 I think I’ll incorporate this idea into my Datadog shard. I was going to force using `Log.context`, but I like this approach a lot better!